src/Network/Network.Test/ScenarioTests/CortexTests.ps1

```diff
@@ -854,7 +854,7 @@ function Test-CortexExpressRouteCRUD
 function Test-VHubRouteTableCRUD {
```

Comment:
please re-record test after modification

Reply:
yes, currenlty recording the test.